packages/react-scripts/template/README.md

```diff
@@ -175,12 +175,14 @@ You don’t have to ever use `eject`. The curated feature set is suitable for sm
 
 ## Supported Language Features and Polyfills
 
-This project supports a superset of the latest JavaScript standard, which includes async and await.  
-In addition to ES2017, it also supports:
+This project supports a superset of the latest JavaScript standard.  
+In addition to [ES6](https://leanpub.com/understandinges6/read) syntax features, it also supports:
```

Comment:
I think a more condensed resource would be more appropriate here, despite the fact that these are the first two search results for ES6 (I'm not trying to be lazy), I found them to be indispensable resources when I was first learning ES6.

interactive:
http://es6-features.org/

document:
https://github.com/lukehoban/es6features